samples/subscriptions.js

```diff
@@ -100,28 +100,34 @@ async function createFlowControlledSubscription(
   // const topicName = 'my-topic';
   // const subscriptionName = 'my-sub';
   // const maxInProgress = 5;
-  // const maxBytes = 10000;
+  // const timeout = 10;
 
   const topic = pubsub.topic(topicName);
 
   const options = {
     flowControl: {
-      maxBytes: maxBytes,
       maxMessages: maxInProgress,
     },
   };
 
+  // Creates a reference to a subscription
```

Comment:
"References an existing subscription" could be better for further clarity and doesn't use the word "create".

Reply:
Since it's possible to reference a non-existing subscription, and get create by setting `autoCreate: true`. I'm going to just say "References a subscription".